client/src/main.rs

```diff
@@ -91,7 +94,18 @@ async fn main() -> Result<()> {
 
     if cli.daemon {
         info!("starting daemon");
-        let indexer = Indexer::bootstrap().await?;
+        let server = "http://localhost:8090";
```

Comment:
You forgot to implement env var here no?

Reply:
fixed